dask/base.py

```diff
@@ -106,15 +107,15 @@ def persist(self, **kwargs):
 
         Parameters
         ----------
-        get : callable, optional
-            A scheduler ``get`` function to use. If not provided, the default
-            is to check the global settings first, and then fall back to
-            the collection defaults.
+        scheduler: string, optional
+            Which scheduler to use like "threads", "synchronous" or "processes".
+            If not provided, the default is to check the global settings first,
+            and then fall back to the collection defaults.
```

Comment:
It also might be nice to support passing in a distributed client directly. This is what I did for `dask-searchcv`:

https://github.com/dask/dask-searchcv/blob/master/dask_searchcv/model_selection.py#L669-L697 and https://github.com/dask/dask-searchcv/blob/master/dask_searchcv/model_selection.py#L992-L997

Note that the string names there were chosen to match joblib/sklearn conventions - I think I prefer the ones you've provided here better.